docs/bridge/bridging-guide.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Use Vara Bridge
+sidebar_position: 2
+sidebar_label: Use Vara Bridge
+---
+
+# Use Vara Bridge
+This page provides a guide on bridging supported VFT tokens from Vara Network to their corresponding wrapped ERC-20 tokens on Ethereum, and vice versa.
+
+## Prerequisites
+
+- A Substrate wallet (for example, **SubWallet**)
+- An Ethereum wallet (for example, **MetaMask**)
+
+---
+
+## Bridging from Vara Network to Ethereum
+
+:::tip
+This tutorial uses Vara's Testnet and **VARAT** tokens. You can acquire a test balance by following the steps [here](docs/idea/account.md#get-test-balance).
+:::
+
+![vara-bridge](img/vara_bridge_tutorial_snap01.png)
+
+1. **Connect Your Wallet**  
+   Connect your Substrate wallet to the Vara Bridge.
+
+2. **Select the Token to Transfer**  
+   Choose **VARAT** from the dropdown menu and specify the number of tokens to transfer.
```

Comment:
once we announce bridge testnet we provide correct token tickers - TVARA, ETH, USDC etc, not as now.